substratevm/src/com.oracle.svm.test.jdk11/src/com/oracle/svm/test/jdk11/AbstractClassSerializationTest.java

```diff
@@ -0,0 +1,42 @@
+package com.oracle.svm.test.jdk11;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.Map;
+
+public class AbstractClassSerializationTest {
+    private static byte[] serializedObject;
+    private static Map<Integer, String> map = Map.of(1, "Test", 2, "This is a test");
+
+    static {
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+        ObjectOutputStream objectOutputStream = null;
+        try {
+            objectOutputStream = new ObjectOutputStream(byteArrayOutputStream);
+            objectOutputStream.writeObject(map);
+            objectOutputStream.flush();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        serializedObject = byteArrayOutputStream.toByteArray();
```

Comment:
I like the idea with baking-in the serialized form at built-time :slightly_smiling_face:

Reply:
Thanks. :)